### ISSUE TITLE
"no process" Error in release built

### ISSUE THREAD

**Issue by dersmon** [dannote/tongue#2]
Disclaimer: I am relatively new to Elixir, so this may either be some missing information in your README for newbies like me, or a real bug. :wink: 

When using Tongue in development (mix run), everything works as expected. 

But now that I tried building my application using [releases](https://github.com/dainst/argos/blob/7dc1f5784f9664275a59a8e1d49932029c893495/DockerfileAggregation), everytime I try to call [Tongue.detect/1](https://github.com/dainst/argos/blob/main/apps/argos_aggregation/lib/argos_aggregation/natural_language_detector.ex#L9), i get the following error:

```bash
06:17:09.743 [error] Task #PID<0.185.0> started from #PID<0.93.0> terminating
** (stop) exited in: GenServer.call(Tongue.Detector, {:detect, "Unter Helden und Göttern. Eduardo Paolozzi und junge Bildhauer der Münchner Akademie in der Glyptothek. Ausstellung München 9. November 1988 - 22. Januar 1989."}, 5000)
    ** (EXIT) no process: the process is not alive or there's no process currently associated with the given name, possibly because its application isn't started
    (elixir 1.11.3) lib/gen_server.ex:1017: GenServer.call/3
    lib/argos_aggregation/natural_language_detector.ex:9: ArgosAggregation.NaturalLanguageDetector.get_language_key/2
    lib/argos_aggregation/data_sources/bibliography.ex:203: ArgosAggregation.Bibliography.BibliographyParser.parse_record/1
    (elixir 1.11.3) lib/task/supervised.ex:90: Task.Supervised.invoke_mfa/2
    (elixir 1.11.3) lib/task/supervised.ex:35: Task.Supervised.reply/5
    (stdlib 3.14) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Function: #Function<5.67668276/0 in ArgosAggregation.Bibliography.DataProvider.process_batch_query/1>
    Args: []

```

**Comment by dannote** [comment]
Did you add `:tongue` to your `applications` in `mix.exs`?

**Comment by dersmon** [comment]
I tried adding `:tongue` to my `extra_applications`, and also added a `required Tongue` to the module that uses Tongue, but to no avail.

**Comment by dersmon** [comment]
Still the same error:

```bash
06:37:12.104 [error] Task #PID<0.238.0> started from #PID<0.93.0> terminating
** (stop) exited in: GenServer.call(Tongue.Detector, {:detect, "Arte rupestre nelle Alpi occidentali dalla Valle Po alla Valchiusella : mostra Torino, 6 novembre 1987 - 24 gennaio 1988."}, 5000)
    ** (EXIT) no process: the process is not alive or there's no process currently associated with the given name, possibly because its application isn't started
    (elixir 1.11.3) lib/gen_server.ex:1017: GenServer.call/3
    lib/argos_aggregation/natural_language_detector.ex:10: ArgosAggregation.NaturalLanguageDetector.get_language_key/2
    lib/argos_aggregation/data_sources/bibliography.ex:203: ArgosAggregation.Bibliography.BibliographyParser.parse_record/1
    (elixir 1.11.3) lib/task/supervised.ex:90: Task.Supervised.invoke_mfa/2
    (elixir 1.11.3) lib/task/supervised.ex:35: Task.Supervised.reply/5
    (stdlib 3.14) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Function: #Function<5.67668276/0 in ArgosAggregation.Bibliography.DataProvider.process_batch_query/1>
    Args: []
** (EXIT from #PID<0.93.0>) exited in: GenServer.call(Tongue.Detector, {:detect, "Nuove acquisizioni del lapidario Zeri di Mentana "}, 5000)
    ** (EXIT) no process: the process is not alive or there's no process currently associated with the given name, possibly because its application isn't started
```

**Comment by dersmon** [comment]
Hey! I managed to get it to work. See: https://elixirforum.com/t/no-process-error-for-dependency-only-in-release-build-umbrella-project/41553 / https://github.com/dainst/argos/blob/56feab0607412a93c5d64473bd5bb386f8774644/mix.exs

Before I only tried adding `:tongue` to the app's `:extra_applications`. Adding `:tongue` to the top level `mix.exs` did the trick. I still am not quite sure why Tongue needs this extra configuration, but thank you for writing the library. :1st_place_medal:
